chain-abci/tests/punishment.rs

```diff
@@ -70,6 +71,34 @@ fn begin_block_should_jail_byzantine_validators() {
     assert_eq!(0, response_end_block.validator_updates.to_vec()[0].power);
 }
 
+#[test]
+fn begin_block_should_ignore_old_byzantine_evidence() {
+    // Init Chain
+    let (env, storage) = ChainEnv::new(Coin::max(), Coin::zero(), 1);
+    let mut app = env.chain_node(storage);
+    let _rsp_init_chain = app.init_chain(&env.req_init_chain());
+
+    // Begin Block
+    let mut req = RequestBeginBlock {
+        byzantine_validators: vec![env.byzantine_evidence(0)].into(),
+        ..env.req_begin_block(1, 0)
+    };
+    req.header.as_mut().unwrap().time = Some(Timestamp {
+        seconds: 172801,
```

Comment:
const MAX_EVIDENCE_VALUE= 172_800
MAX_EVIDENCE_VALUE+1?

Reply:
Changed to fetch the value from `app`.